package.json

```diff
@@ -5,12 +5,14 @@
   "main": "lib/index.js",
   "scripts": {
     "lint": "./node_modules/.bin/standard --verbose | ./node_modules/.bin/snazzy",
+    "build": "yarn build-es5 && yarn build-dist && yarn build-dist-prod",
```

Comment:
Ah so of course it because our release process currently depends on `npm`. This was written at about 3am. I'll update it.